app/dashboard/templates/bounty/details.html

```diff
@@ -93,7 +93,7 @@ <h4 id="title" class="font-title"></h4>
                 </div>
 
                 <div>
-                  <span class="bounty-info-heading">{% trans "Project Length" %}</span>
+                  <span class="bounty-info-heading">{% trans "Time Commitment" %}</span>
```

Comment:
We'll want to run `make autotranslate` or `makemessages`/`compilemessages` via manage.py following changes like this. @arun1595 @thelostone-mc

Reply:
my bad ! totally slipped my mind